docs/presentation/index.md

```diff
@@ -2,11 +2,11 @@
 
```

Comment:
When I click on Specifications item under Hierarchies sidebar section, I get redirected to Child Node Rule page.

Reply:
correct, we don't have a separate page for specifications, so we link to a specifications' section in child node rule page